src/components/site/SiteStatus.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const SiteStatus = props => <p>
+                            <span className="status-text">{props.StatusName}</span>
```

Comment:
So new thing here, since we build this component, and we have a required prop "status" we want to add proptypes to this file. Checkout [checkbox](https://github.com/baltimorecounty/dotgov-components/blob/integration/src/components/basic/form/Checkbox.js) for an example. This will also populate the docs.

Reply:
oh cool, will add this to all the components with props